### PR TITLE
Add new Instant Answer: Julia-Lang packages

### PR DESCRIPTION
**Pull request by cskksc**
**What does your Instant Answer do?**
Displays information about julia lang packages.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
It reduces the effort for a developer to go to the website and check the details.

**What is the data source for your Instant Answer? (Provide a link if possible)**
http://pkg.julialang.org/

**Why did you choose this data source?**
It is the official source from julia-lang and I assume would be updated regularly.

**Are there any other alternative (better) data sources?**
None that i know of.

**What are some example queries that trigger this Instant Answer?**
"julia json" or "julia arduino"

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Julia lang programmers.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Yes. [IA idea](https://duck.co/ideas/idea/5999/julia-packages). But I am the one who created it, so not sure if it counts.

**Which existing Instant Answers will this one supercede/overlap with?**
Not sure.

**Are you having any problems? Do you need our help with anything?**
I cant test fathead type IA's. Any help regarding that would be great.

**Where did you hear about DuckDuckHack? (For first time contributors)**
Friends.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
None.

https://duck.co/ia/view/julia

### CODE REVIEW

**Comment by moollaza** [comment]
@cskksc thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

**Comment by jdorweiler** [comment]
@cskksc We have ruby 1.8.7 and I'm getting this error: 
```
sudo gem install nokogiri                                                                   
ERROR:  Error installing nokogiri:
        nokogiri requires Ruby version >= 1.9.2.
```
Could you change your parse script to work with 1.8.7 or should I look into upgrading ruby?

**Comment by cskksc** [comment]
We can install an older version of nokogiri by ```gem install nokogiri -v 1.5.10```, which is compatible with ```ruby-1.8.7```. I have verified that this version of ```nokogiri``` doesn't break anything.

**Comment by jdorweiler** [comment]
@cskksc thanks, that worked but I'm getting this error when trying to run `parse.sh`
```
parse.rb:3:in `require': no such file to load -- nokogiri (LoadError)
        from parse.rb:3
```

**Comment by cskksc** [comment]
@jdorweiler Sorry for all the trouble. I have re-written the parsing script in python, using Beautiful Soup. This is used by many other IA's and shouldn't cause any problems with the dependencies. I'm not exactly sure why we're getting an error with nokogiri.

File: share/fathead/julia/parse.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python27 parse.py
```
Comment (jdorweiler) [inline share/fathead/julia/parse.sh]: Could you change to python2.7

**Comment by jdorweiler** [comment]
@cskksc Sorry we haven't had a fathead done in Ruby in a while so I think our install is a bit out of date.  Your python script works great though.  Just one small comment thing I found. 
Let me know what you think:  https://ddh10.duckduckgo.com/?q=julia+aws&ia=about

You can see the trigger words I added on the IA page https://duck.co/ia/view/julia
Any other trigger words that you would like?

Could you remove the version number?  I think that's likely to go out of date frequently.

**Comment by cskksc** [comment]
@jdorweiler It looks awesome :smiley:. I cannot think of any other triggers to add. Those would cover almost everything I think. Also I had messed up the redirect links. [Eg](https://ddh10.duckduckgo.com/?q=julialang+abstract+domains). Had missed the the 3rd param, which decides where to redirect. Could we please deploy again to check if it is working now ?

**Comment by jdorweiler** [comment]
@cskksc you will to split the redirect entries by tabs just like the normal abstract entries.

**Comment by cskksc** [comment]
Yeah, that also was wrong. Have fixed it.

**Comment by jdorweiler** [comment]
@cskksc looks good now, thanks. 
https://ddh10.duckduckgo.com/?q=julialang+abstract+domains&ia=about

**Comment by cskksc** [comment]
@jdorweiler Looks cool. When are we planning to deploy into production ? Are there some fixed release cycles, or more tests ?

**Comment by jdorweiler** [comment]
@cskksc we're actually working on a new way for deploying fatheads so it might be another week or two.  Could be earlier but I'll ping you when it's ready to go.

**Comment by cskksc** [comment]
@jdorweiler hey, when is this planned to put into production.

**Comment by jdorweiler** [comment]
@cskksc sorry for the delay.  Looks like we will be ready for it next week.

**Comment by cskksc** [comment]
Its okay. Just curious. PS. its my first contribution. :smiley:

**Comment by jdorweiler** [comment]
@cskksc I'll ping you when it's live but we're ready to go now!  :cake:

**Comment by jdorweiler** [comment]
@cskksc https://duckduckgo.com/?q=julia+aws
Sorry it took so long!

**Comment by cskksc** [comment]
@jdorweiler Not at all. It looks nice. Thanks a lot. :)

**Comment by cskksc** [comment]
@jdorweiler Hey, I've started a thread on [julia mailing list](https://groups.google.com/forum/#!forum/julia-users). Can you please help by sharing some information there ? Here's an interesting [one](https://groups.google.com/d/msg/julia-users/M95_eyCl08Y/SQAdUFwLFwAJ).

**Comment by jdorweiler** [comment]
@cskksc thanks!  Sounds like people like the IA :smile: 
The query with "images" is going to be tricky since it's generally a strong indication that someone is looking for actual images.

**Comment by moollaza** [comment]
@cskksc congrats! It's live: https://duckduckgo.com/?q=julia+arduino&ia=about

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**